docs/DeveloperGuide.md

```diff
@@ -1,34 +1,513 @@
 # Developer Guide
 
-## Design & implementation
+## 1. Table of content
```

Comment:
Slight error here: Table of contents instead of content.
Do make sure the links work as well!